config/docker-compose.yml

```diff
@@ -0,0 +1,53 @@
+version: "3.7"
+services:
+  db:
+    image: "mysql:5.7"
+    restart: always
+    environment:
+     MYSQL_ROOT_PASSWORD: root
+     MYSQL_USERNAME: root
+     MYSQL_PASSWORD: root
+        - '3306:3306'
```

Comment:
What's this?

Reply:
this should be the port number. Because Port 3306 is usually the default port for the classic MySQL protocol

Reply:
Add a line somewhere in the documentation to make this assumption explicit.  E.g., 

> We use 3306 as the default port for the MySQL configuration as this is the default by convention.